plugins/LadspaEffect/calf/src/synth.cpp

```diff
@@ -25,7 +25,7 @@ using namespace std;
 
 void basic_synth::kill_note(int note, int vel, bool just_one)
 {
-    for (list<dsp::voice *>::iterator it = active_voices.begin(); it != active_voices.end(); it++) {
```

Comment:
On 01/09/2015 09:38 PM, Tres Finocchiaro wrote:
>
> In plugins/LadspaEffect/calf/src/synth.cpp
> <https://github.com/LMMS/lmms/pull/1585#discussion-diff-22737757>:
>
> > @@ -25,7 +25,7 @@ using namespace std;
> >  
> >  void basic_synth::kill_note(int note, int vel, bool just_one)
> >  {
> > -    for (list<dsp::voice *>::iterator it = active_voices.begin(); it != active_voices.end(); it++) {
>
> Hmm... @Lukas-W <https://github.com/Lukas-W> @diizy
> <https://github.com/diizy> Are these old calf plugins our problem now?
> I see some tweaks are being made, but upstream has diverged too much
> in this case, right?
>

That looks like Calf synth plugin code which doesn't work in any LMMS
plugins, probably DSSI/LV2 stuff. Touching it at all is pointless.